pyuvdata/uvdata.py

```diff
@@ -1751,7 +1894,11 @@ def read_uvfits(self, filename, antenna_nums=None, antenna_names=None,
                                     run_check=run_check, check_extra=check_extra,
                                     run_check_acceptability=run_check_acceptability,
                                     keep_all_metadata=keep_all_metadata)
-                    self += uv2
+                    if axis is not None:
+                        self.fast_concat(uv2, axis, run_check=True, check_extra=True,
+                                         run_check_acceptability=True, inplace=True)
```

Comment:
Should the check keywords be based on what was passed into this function?

Reply:
yes, thank you!